### PR TITLE
Fix rendering HTML elements when updating unique flash messages

### PR DESCRIPTION
**Pull request by Martchus**
This especially concerns the flash messages shown within the developer
mode panel, see https://progress.opensuse.org/issues/104917.

### CODE REVIEW

**Review by okurz — APPROVED** [review]

File: assets/javascripts/openqa.js [hunk 1]
```diff
@@ -58,9 +58,10 @@ function addUniqueFlash(status, id, text, container) {
     window.uniqueFlashMessages = {};
   }
   // update existing flash message
-  var existingFlashMessage = window.uniqueFlashMessages[id];
+  const existingFlashMessage = window.uniqueFlashMessages[id];
   if (existingFlashMessage) {
-    existingFlashMessage.find('span').first().text(text);
+    const element = typeof text === 'string' ? $('<span>' + text + '</span>') : text;
```
Comment (kraih) [inline assets/javascripts/openqa.js:63]: Why is there a `typeof text === 'string'` check? I went through the calls of this function and didn't see a case where it was not a `string`. Also, why is the element created with `$()`?

Comment (Martchus) [reply assets/javascripts/openqa.js:63]: All is just in consistency with the normal `addFlash`.

Comment (kraih) [reply assets/javascripts/openqa.js:63]: At least the `$(...)` doesn't do anything at all i think.

Comment (Martchus) [reply assets/javascripts/openqa.js]: I removed the `$(…)` in both functions.

**Review by okurz — APPROVED** [review]

**Review by kraih — APPROVED** [review]
Creative solution with `makeFlashElement`, i like it. 😁

**Pull request merged**